### PR TITLE
feat: add docker.io to image

### PR DESCRIPTION
**Pull request by pichoemr**
#__ 

## Checklist
- [ ] I didn't over-scope my PR
- [ ] My PR title matches the [commit convention](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] I did not include breaking changes
- [ ] I included unit tests that cover my changes
- [ ] I added/updated the documentation about my changes
- [ ] I made my own code-review before requesting one

### CODE REVIEW

File: lenra.yml [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-    - "!/services/"
-    - "!/widgets/"
+      - "**"
+      - "!/*.json"
+      - "!/server.js"
+      - "!/index.js"
+      - "!/listeners/"
+      - "!/resources/"
+      - "!/services/"
+      - "!/widgets/"
```
Comment (shiipou) [inline lenra.yml:26]:
```suggestion
    - "**"
    - "!/*.json"
    - "!/server.js"
    - "!/index.js"
    - "!/listeners/"
    - "!/resources/"
    - "!/services/"
    - "!/widgets/"
```

File: lenra.yml [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
     root:
       script:
-      - chown -R 1000 /app
+        - chown -R 1000 /app
     adds:
-    - .
+      - .
     script:
-    - npm i --omit=dev
+      - npm i --omit=dev
```
Comment (shiipou) [inline lenra.yml:12]:
```suggestion
    - npm i --omit=dev
```

File: lenra.yml [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
-    from: "bitnami/node:16"
+    from: "docker.io/bitnami/node:16"
     workdir: /app
     root:
       script:
-      - chown -R 1000 /app
+        - chown -R 1000 /app
     adds:
-    - .
+      - .
```
Comment (shiipou) [inline lenra.yml:10]:
```suggestion
    - .
```

File: lenra.yml [hunk 4]
```diff
@@ -1,26 +1,26 @@
 componentsApi: "1.0"
 generator:
   dofigen:
-    from: "bitnami/node:16"
+    from: "docker.io/bitnami/node:16"
     workdir: /app
     root:
       script:
-      - chown -R 1000 /app
+        - chown -R 1000 /app
```
Comment (shiipou) [inline lenra.yml:8]:
```suggestion
      - chown -R 1000 /app
```

**Review by shiipou — CHANGES_REQUESTED** [review]
That's not much, but there's no point in pushing an indentation change.

**Review by jonas-martinez — APPROVED** [review]

**Comment by pichoemr** [comment]
@shiipou it is vs code which makes the automatic indent, the indentation will be in each PR which touches yml, maybe let it

**Review by shiipou — APPROVED** [review]

**Pull request merged**